shared/fixes/ansible/dconf_gnome_login_banner_text.yml

```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- (xccdf-var login_banner_text)
+
+- name: "@RULE_TITLE@"
+  ini_file:
+    dest: /etc/dconf/db/gdm.d/00-security-settings
+    section: org/gnome/login-screen
+    option: banner-message-text
+    value: string '{{ login_banner_text }}'
+    create: yes
+  tags:
+    @ANSIBLE_TAGS@
+
+- name: "Prevent user modification of the GNOME3 Login Warning Banner Text"
+  lineinfile:
+    path: '/etc/dconf/db/gdm.d/locks/00-security-settings-lock'
```

Comment:
What if the file / directory does not exist? I got an Ansible error when I tried to apply this task.

Reply:
I didn't realize that you could have a GDM login without the gdm.d directory. I've never seen an install without it so I just assumed it would be there. I can add a task to make sure it exists.

Reply:
Ah I just checked my Fedora machine and it doesn't have a gdm directory. I'll add that stuff in.